packages/gatsby-theme-vtex/src/components/ProductImageGallery/Miniatures/Page.tsx

```diff
@@ -0,0 +1,19 @@
+import React, { FC } from 'react'
+import { AspectImage, YoutubeThumb } from '@vtex/store-ui'
+
+import { Item } from '../Page'
+
+interface Props {
+  item: Item
+  variant: string
+}
+
+const Page: FC<Props> = ({ item, variant }) => {
+  if (item.type === 'image') {
+    return <AspectImage ratio={1} {...item.props} {...item.props.targetProps} />
+  }
+
+  return <YoutubeThumb src={item.props.src} variant={variant} />
```

Comment:
Add a todo to turn this into `VideoThumb`. Our Admin supports adding videos from Vimeo, for example.

That's why I've created the VideoIframe wrapper (despite only supporting Youtube now)

Reply:
Cool ! I'm just not sure if we should suport it via props or via shadowing. I think shadowing is better, but I really don't know. I will add the comment though